ticdc: update tiup deployment prompt and filter rule link (#3651)
pingcap/docs: ticdc/ticdc-overview.md

- The [TiKV Hibernate Region](https://github.com/tikv/tikv/blob/master/docs/reference/configuration/raftstore-config.md#hibernate-region). TiCDC prevents the Region from entering the hibernated state.
- The scheduling of existing replication tables to new TiCDC nodes, after the capacity of the TiCDC cluster is scaled out.

## Install and deploy TiCDC 

> TiCDC supports deploying TiCDC components while deploying a new TiDB cluster using TiUP. You only need to [add the TiCDC](/production-deployment-using-tiup.md#step-3-edit-the-initialization-configuration-file) to the configuration file when TiUP starts the TiDB cluster.

Reviewer: ```suggestion
You can deploy TiCDC components in the process of deploying a new TiDB cluster using TiUP. You only need to [add TiCDC](/production-deployment-using-tiup.md#step-3-edit-the-initialization-configuration-file) to the configuration file when TiUP starts the TiDB cluster.
```

> Currently, TiCDC supports adding TiCDC components to the TiDB cluster using either TiUP or Binary. For details, see [Deploy and install TiCDC](/ticdc/manage-ticdc.md#deploy-and-install-ticdc).

Reviewer: `TiCDC supports adding TiCDC components` does not make sense.
```suggestion
Currently, you can also add TiCDC components to an existing TiDB cluster using either TiUP or binary. For details, see [Deploy and install TiCDC](/ticdc/manage-ticdc.md#deploy-and-install-ticdc).
```

## Manage TiCDC Cluster and Replication Tasks

> Currently, TiCDC supports using `cdc cli` to manage a TiCDC cluster and data replication tasks. For details, see: 

Reviewer: ```suggestion
Currently, you can use the `cdc cli` tool to manage the status of a TiCDC cluster and data replication tasks. For details, see: 
```
